lib/hypervisor/hv_kvm/__init__.py

```diff
@@ -2555,13 +2555,14 @@ def MigrateInstance(self, cluster_name, instance, target, live_migration):
     if not live_migration:
       self._CallMonitorCommand(instance_name, "stop")
 
-    migrate_command = ("migrate_set_speed %dm" %
-                       instance.hvparams[constants.HV_MIGRATION_BANDWIDTH])
-    self._CallMonitorCommand(instance_name, migrate_command)
-
-    migrate_command = ("migrate_set_downtime %dms" %
-                       instance.hvparams[constants.HV_MIGRATION_DOWNTIME])
-    self._CallMonitorCommand(instance_name, migrate_command)
+    qmp = QmpConnection(self._InstanceQmpMonitor(instance_name))
```

Comment:
Minor, but there's the `@_with_qmp` decorator that makes a QMP connection available at `self.qmp`. This will be especially handy if you decide to switch everything over to QMP.